canvas_account_admin_tools/requirements/base.txt

```diff
@@ -37,3 +39,6 @@ redis==3.3.8
 requests==2.22.0
 simplejson==3.10.0
 git+ssh://git@github.com/Harvard-University-iCommons/django-ssm-parameter-store.git@v0.3#egg=django-ssm-parameter-store==0.3
+
+splunk_handler==2.2.0
+python-json-logger==0.1.11
```

Comment:
Any reason we're not using a newer versions?
* https://github.com/zach-taylor/splunk_handler/releases
* https://github.com/madzak/python-json-logger/blob/master/CHANGELOG.md